var/spack/repos/builtin/packages/qtgraph/package.py

```diff
@@ -0,0 +1,85 @@
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+##############################################################################
+# Copyright (c) 2015-2017 Krell Institute. All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation; either version 2 of the License, or (at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+# Place, Suite 330, Boston, MA  02111-1307  USA
+##############################################################################
+from spack import *
+import os
+
+
+class Qtgraph(QMakePackage):
+    """The baseline library used in the CUDA-centric Open|SpeedShop Graphical
+       User Interface (GUI) which allows Graphviz DOT formatted data to be
+       imported into a Qt application by wrapping the Graphviz libcgraph and
+       libgvc within the Qt Graphics View Framework."""
+
+    # url for qtgraph package's homepage here.
+    homepage = "https://github.com/OpenSpeedShop/QtGraph"
+
+    # Use when the git repository is available
+    url      = "https://github.com/OpenSpeedShop/QtGraph.git"
+    version('1.0.0', branch='master',
+            git='https://github.com/OpenSpeedShop/QtGraph.git')
+
+    # qtgraph depends on these packages
+    depends_on('qt@4.8.6:')
+    depends_on('graphviz')
+
+    def setup_environment(self, spack_env, run_env):
+        """Set up the compile and runtime environments for a package."""
+        spack_env.set('GRAPHVIZ_ROOT', self.spec['graphviz'].prefix)
+        spack_env.set('INSTALL_ROOT', self.prefix)
+
+        # The assumed qt version changed to QT5 (as of paraview 5.2.1),
+        # so explicitly specify which QT major version is actually being used
+
+        # What lib suffix do we need to use?
+        libdir = self.prefix.lib64
+        if not os.path.isdir(libdir):
+            libdir = self.prefix.lib
```

Comment:
I prefer `prefix.lib`, but it depends on where the package decides to install to. In any case, it probably isn't dynamic. See which one it is using and stick with it.